database/pgsql/testdata/data.sql

```diff
@@ -2,7 +2,8 @@
 INSERT INTO namespace (id, name, version_format) VALUES
   (1, 'debian:7', 'dpkg'),
   (2, 'debian:8', 'dpkg'),
-  (3, 'fake:1.0', 'rpm');
+  (3, 'fake:1.0', 'rpm'),
+  (4, 'cpe:/o:redhat:enterprise_linux:7::server', 'rpm');
```

Comment:
Does 'cpe:/o:redhat:enterprise_linux:7::server' has any version information?

If this is the actual format for CPE, I'll have to change the namespace db a bit because right now we assume namespace name consists of two components: `<name>:<version>` as a string.

Reply:
Also, please rebase to master, and test. We found a bug in our tests that prevents database test failure from showing up.

Reply:
The version is usually included in CPE (in this case it is 7), but from the definition of CPE the version can be omitted.

https://access.redhat.com/blogs/766093/posts/2387501

Reply:
Yes, I noticed the bug. Today I spend a few hours trying to figure out why my test is not failing and after I found the cause and I fixed it I noticed that you fixed it yesterday